server/reviewsController.js

```diff
@@ -0,0 +1,192 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable guard-for-in */
+/* eslint-disable no-shadow */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-console */
+const { db } = require('../database/index');
+
+const reviews = (req, res) => {
+  const { productId } = req.params;
+
+  const query = `
```

Comment:
this is a really nice multi query statement.  Ima come back to this and review it for my component! 🔥